java/src/com/ibm/streamsx/topology/builder/GraphBuilder.java

```diff
@@ -246,11 +247,30 @@ public JSONObject complete() {
         return ops;
     }
 
-    public void createSubmissionParameter(String name, JSONAble jsonable) {
+    /**
+     * Create a submission parameter.
+     * <p>  
+     * The SubmissionParameter parameter value json is: 
+     * <pre><code>
+     * object {
+     *   type : "submissionParameter"
+     *   value : object {
+     *     name : string. submission parameter name
+     *     valueClassName : string. a java type
+     *     typeModifier : optional string to modify interpretation of
```

Comment:
Can't we just use the Java Operator API metatypes here, rather than another type scheme?